src/tests/clipperUI/components/previewViewer/augmentationPreview_tests.tsx

```diff
@@ -29,13 +25,64 @@ export class AugmentationPreviewTests extends TestModule {
 	}
 
 	protected tests() {
+		function isPositive(value: number): number {
```

Comment:
This is not needed:
a) You can just assert n > 0 in the test itself
b) Why does this return a number and not a boolean?
c) 0 is a valid positive tabindex but is treated as negative here